romancal/pipeline/exposure_pipeline.py

```diff
@@ -183,24 +190,46 @@ def process(self, input):
         #          observations. This should not occur on-prem
         if result.meta.exposure.type == "WFI_IMAGE":
             if file_type == "asdf":
+                result.meta.cal_step.tweakreg = "SKIPPED"
+                # mc_result = ModelContainer(result)
                 mc_result = self.tweakreg([result])
-                result = mc_result._models.pop()
+                if hasattr(ModelContainer(result), "_models") and mc_result._models:
+                    result = mc_result._models.pop()
+                else:
+                    result.meta.cal_step.tweakreg == "SKIPPED"
+
             if file_type == "asn":
                 result = self.tweakreg(tweakreg_input)
+                for model in result._models:
+                    model.meta.cal_step.tweakreg = "SKIPPED"
```

Comment:
I think I expected COMPLETE here?

Reply:
When tweakreg is run with the option "--steps.tweakreg.skip=True" it is not honoring that and setting the step to skipped. So for the test RCAL-665 to pass this needs to be manually set to "SKIPPED".

This should not remain in place. My plan is to write a function that will check the skip flag that is passed to the step and set it appropriately and use this for all the pipeline steps. I think we should have a uniform method. I checked JWST and they seem to handle this flag in a non-uniform manner and I don't want to reproduce that. I'd like to fix this after B12.

Reply:
Sounds good, thanks.